spm/doc/INTRO.md

```diff
@@ -0,0 +1,117 @@
+## The Secure Partition Manager
+
+The **Secure Partition Manager (SPM)** is a Platform Security Architecture (PSA) compliant software hypervisor that creates and manages independent Secure Partitions on Arm Cortex&reg;-M microcontrollers. It increases resilience against malware and protects secrets from leaking between different modules in the same application. The SPM complements other important security features, such as safe firmware updates and secure crypto libraries.
+
+The SPM provides hardware-enforced partitions for individual code blocks by limiting access to memories and peripherals using the existing hardware security features of the Cortex&reg;-M microcontrollers. It isolates software in partitions, managing the execution of software within those partitions, and providing Inter Process Communication (IPC) between the partitions. Correct use of SPM prevents malware from becoming resident on the device and enables protection of device secrets, such as cryptographic keys.
+
+### Isolating partitions in the Secure Processing Environment
+
+The SPM and the Secure Partitions are located in the Secure Processing Environment (SPE), isolating them from the Non-Secure Processing Environment (NSPE), which contains the application firmware, OS kernel and libraries, and other non-secure hardware resources.
+
+A Secure Partition is a container for one or more Secure Functions, and a platform may have multiple Secure Partitions. Secure Partitions provide the execution environment for security functionality.
```

Comment:
Please lower case "secure partition" and "secure functions" and "secure partitions".